addon/components/base/bs-modal.js

```diff
@@ -418,7 +418,9 @@ export default Component.extend(TransitionSupport, {
       this.set('inDom', true);
       schedule('afterRender', () => {
         let modalEl = this.get('modalElement');
-        modalEl.scrollTop = 0;
```

Comment:
Why did you remove this line? Shouldn't it just be moved after the guard you added?

Reply:
Oops - that was an accident. Updated.